### PR TITLE
chore: remove duplicate code, add docs to helper

### PR DESCRIPTION
**Pull request by avodovnik**

### CODE REVIEW

**Review by kblok — APPROVED** [review]

**Comment by pavelfeldman** [comment]
Sorry, my intention was the opposite - I don't think we should offer our customers this patched method. So I would just delete the one in PlaywrightTest.

**Comment by avodovnik** [comment]
> Sorry, my intention was the opposite - I don't think we should offer our customers this patched method. So I would just delete the one in PlaywrightTest.

Oh! All the code right now uses that (in our tests). I'll move this back to the other file, then, and mark internal. :-)

**Comment by kblok** [comment]
conflicted @avodovnik

**Comment by avodovnik** [comment]
> conflicted @avodovnik

Yes, not ready to be merged yet. I'll mark it as draft for now, to avoid a mishap.

**Review by pavelfeldman — APPROVED** [review]

File: src/Playwright.Tests/BaseTests/PlaywrightAssert.cs [hunk 1]
```diff
@@ -46,7 +49,7 @@ public static class TestExtensions
             }
         }
 
-        public static void DebugLog(this PlaywrightTest test, string text)
+        internal static void DebugLog(string text)
         {
             TestContext.Progress.WriteLine(text);
         }
```
Comment (kblok) [inline src/Playwright.Tests/BaseTests/PlaywrightAssert.cs:55]:
```suggestion
        internal static void DebugLog(string text) => TestContext.Progress.WriteLine(text);
```

**Review by kblok — APPROVED** [review]

File: src/Playwright.Tests/.runsettings [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<RunSettings>
+   <NUnit>
+       <NumberOfTestWorkers>24</NumberOfTestWorkers>
```
Comment (pavelfeldman) [inline src/Playwright.Tests/.runsettings:3]: I have 5! Let's leave defaults that count number of cores.

**Review by pavelfeldman — APPROVED** [review]

File: src/Playwright.Tests/.runsettings (hunk 2 above)
Comment (avodovnik) [reply src/Playwright.Tests/.runsettings:3]: This one shouldn't have been commited. Removing...

**Pull request merged**